src/app/modules/shared_models/member_model.nim

```diff
@@ -117,6 +119,15 @@ QtObject:
       result = newQVariant(item.pubKey)
     of ModelRole.DisplayName:
       result = newQVariant(item.displayName)
+    of ModelRole.PreferredDisplayName:
```

Comment:
nitpick: I'm wondering if there's a reason why it's not a member in the `item` object? 😄

We'll still need to take care of the  `dataChanged` events for this new role and add this logic in all functions updating any of the user names. So IMO adding it as a member in the `item` object would be a nice way to go here.

Reply:
I prefer adding the `PreferredDisplayName` in `dataChanged` in the model, as it's way simpler than modifying all the modules that use this model. 
Also, this way, the logic for the order which name is more important is only in one place.